vendor/github.com/kubernetes-csi/csi-test/pkg/sanity/controller.go

```diff
@@ -1622,8 +1622,11 @@ var _ = DescribeSanity("CreateSnapshot [Controller Server]", func(sc *SanityCont
 		Expect(snap1).NotTo(BeNil())
 		verifySnapshotInfo(snap1.GetSnapshot())
 
+		volume2, err := c.CreateVolume(context.Background(), MakeCreateVolumeReq(sc, "CreateSnapshot-volume-3"))
+		Expect(err).ToNot(HaveOccurred())
+
 		By("creating a snapshot with the same name but different volume source id")
-		req2 := MakeCreateSnapshotReq(sc, "CreateSnapshot-snapshot-2", "test001", nil)
+		req2 := MakeCreateSnapshotReq(sc, "CreateSnapshot-snapshot-2", volume2.GetVolume().GetId(), nil)
```

Comment:
was this a `dep` update? If this is an error in the sanity package I'd rather we do an update in the package itself then do a dep update here

Reply:
PR to update the sanity test https://github.com/kubernetes-csi/csi-test/pull/116